system_strings/sysmes_text.en

```diff
@@ -984,11 +984,11 @@ I wake up in my room, on my bed^as if nothing had happened. It^was a dream, afte
 It's unusual to see Arihiko in^the morning. When I ask him why^he's acting strangely, he tells^me that Ciel-senpai had come to^see him.
 I don't feel hungry today. When^I tell her that I'm not eating^lunch, an electric current runs^through Ciel――
 I'll go and meet Ciel-senpai.^After preparing the necessary^tribute, I head to the unfamiliar^tea ceremony room.
-現実と認める。^この手で殺めた女を置き去りに、^マンションを飛び出してゆく――
-降り出した雨は容赦なく、^体から温度を奪っていく。^帰る場所はない。
-シエルのアパート。^心に花を。花には水を。
-罪が消える事はない。^それでも、先輩の言葉に^何かが救われた気がした……。
-眠れぬまま、迷っていた。^この先どうするべきなのか。^どうすれば罪を償えるのか。
+I accept it as reality. Leaving^behind the woman I killed with^my own hands, I run out of the^apartment――
+The rain that began to fall was^relentless and saps the warmth,^I have nowhere to go.
```

Comment:
saps _my_ warmth - 体から温度を奪っていく "Saps the heat from my body"

Reply:
Also, JP ～た can be more for aspect than tense - this rain beginning to fall is a 'complete' action, but it's still falling and is still relentless. This can be promoted to present -
> The unrelenting rain is sapping the warmth from my body. I have nowhere to go.